README.md

```diff
@@ -8,55 +8,90 @@
 Stencil Playwright is a testing adapter package that allows developers to easily use the [Playwright testing framework](https://playwright.dev/docs/intro)
 in their Stencil project.
 
-For full documentation, please see the [Playwright testing docs on the official Stencil site](https://stenciljs.com/docs/next/testing/playwright/overview).
+For full documentation, please see the [Playwright testing docs on the official Stencil site](https://stenciljs.com/docs/testing/playwright/overview).
 
 ## Getting Started
 
-1. Install the necessary packages using your preferred package manager:
+1. Install the necessary dependencies:
 
-   ```bash
+   ```bash npm2yarn
```

Comment:
FWIW, I don't think we need the `npm2yarn`, since this isn't rendering in Docusaurus